src/wkt/tests/message_with_string.rs

```diff
@@ -228,4 +228,12 @@ mod tests {
         assert_eq!(output, json!({}));
         Ok(())
     }
+
+    #[test_case(json!({}), "MessageWithString { singular: \"\", optional: None, repeated: [], map_value: {}, map_key: {}, map_key_value: {} }")]
+    #[test_case(json!({"unknown0": 23, "unknown1": "test"}), "MessageWithString { singular: \"\", optional: None, repeated: [], map_value: {}, map_key: {}, map_key_value: {}, _unknown_fields: {\"unknown0\": Number(23), \"unknown1\": String(\"test\")} }")]
+    fn test_debug_trait(input: Value, debug_str: &str) -> Result {
+        let got = serde_json::from_value::<MessageWithString>(input)?;
+        assert_eq!(format!("{got:?}"), debug_str);
```

Comment:
I think this test depends on the order for unknown fields, and since they are in a hashmap, that seems brittle.

Consider testing for "the `_unknown_fields` thing appears in the formatted debug test" and "the `unknown0` field and value also appear", etc.